iquip/apps/builder.py

```diff
@@ -404,6 +478,33 @@ def initSchedOptsEntry(self):
             self.builderFrame.schedOptsListWidget.addItem(item)
             self.builderFrame.schedOptsListWidget.setItemWidget(item, widget)
 
+    @pyqtSlot()
+    def reloadArgs(self):
+        """Reloads the build arguments.
+        
+        Once the reloadArgsButton is clicked, this is called.
+        """
+        self.thread = ExperimentInfoThread(self.experimentPath, self.onReloaded, self)
+        self.thread.start()
+
+    def onReloaded(
+        self,
+        _experimentPath: str,
```

Comment:
Oh did we use an underscore to avoid the "unused argument" warning?
I think we should agree about the convention, because we used the pylint disable comment in #69.

Reply:
Umm.. I think if the method is overridden, it looks better to use the original variable name like #49.
But, in this case where the method isn't overridden but we can't remove the unused variable, in according to [styleguide](https://google.github.io/styleguide/pyguide.html#214-decision), I want to use an underline.

Or, how about defining only `_`, not `_experimentPath`?

Reply:
What do you mean by "like #49"?

As you said, since it is not overridden, we don't have to use the original name.
However, to show the "place-holding role" of the unused variable, I think your current naming is good enough.

Reply:
> What do you mean by "like https://github.com/snu-quiqcl/iquip/issues/49"?

Oh, sorry. I intended #69 you metioned.

Reply:
Oh I see.